app/serializers/admin/auction_serializer.rb

```diff
@@ -2,35 +2,31 @@ class Admin::AuctionSerializer < ActiveModel::Serializer
   has_many :bids, serializer: Admin::BidSerializer
 
   attributes(
-    :issue_url,
-    :github_repo,
-    :start_price,
-    :started_at,
-    :ended_at,
+    :billable_to,
+    :cap_proposal_url,
+    :created_at,
     :delivery_due_at,
     :delivery_url,
-    :cap_proposal_url,
-    :awardee_paid_status,
-    :title,
     :description,
+    :ended_at,
+    :github_repo,
     :id,
-    :created_at,
-    :updated_at,
+    :issue_url,
     :notes,
+    :paid_at,
+    :start_price,
+    :started_at,
     :summary,
-    :billable_to
+    :title,
+    :updated_at
   )
 
-  def delivery_due_at
-    object.delivery_due_at.try(:iso8601)
-  end
-
   def created_at
     object.created_at.iso8601
   end
 
-  def updated_at
-    object.updated_at.iso8601
+  def delivery_due_at
+    object.delivery_due_at.try(:iso8601)
```

Comment:
Not going to affect this PR, but these `try` clauses feel like a smell that would have to be resolved at some point